### PR TITLE
Speeding up Scaper (part one)

### PR DESCRIPTION
**Pull request by pseeth**
This monster of a PR speeds up Scaper considerably. So far it contains the following PRs (which have already been reviewed):

- PR #106 
- PR #110 
- PR #111 
- PR #118

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Changelog
 ---------
```
Comment (justinsalamon) [inline docs/changes.rst:4]: need to pull updates from master which is on 1.3.9 and update changelog with everything in this PR!

Comment (justinsalamon) [reply docs/changes.rst:4]: bump to 1.4.0

File: scaper/version.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 """Version info"""
 
 short_version = '1.3'
-version = '1.3.7'
+version = '1.3.8'
```
Comment (justinsalamon) [inline scaper/version.py:6]: bump to 1.4.0

File: setup.py [hunk 3]
```diff
@@ -36,17 +36,17 @@
             "Programming Language :: Python :: 3.6",
         ],
     install_requires=[
-        'sox>=1.3.3',
+        'sox==1.4.0b0',
```
Comment (justinsalamon) [inline setup.py:39]: sox 1.4.0 has been released - switch to this instead of beta?

**Review by justinsalamon — CHANGES_REQUESTED** [review]

File: setup.py (hunk 3 above)
Comment (pseeth) [reply setup.py:39]: Good catch. Will do!

**Review by justinsalamon — APPROVED** [review]
👍

**Review by justinsalamon — APPROVED** [review]

**Pull request merged**